### ISSUE TITLE
keybase-gui 5.3 broken

### ISSUE THREAD

**Issue by FelschR** [NixOS/nixpkgs#82930]
**Describe the bug**
Since the update to 5.3 (#82740) I'm getting the following error when running keybase-gui:
```
segmentation fault (core dumped)  keybase-gui
```

It's apparently an upstream issue: https://github.com/keybase/client/issues/23018

**To Reproduce**
Update to the latest keybase-gui version on `nixos-unstable` and try to run it.

**Expected behavior**
keybase-gui should start without error.

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.5.9, NixOS, 20.09pre217446.ddf87fb1baf (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.3`
 - channels(root): `"nixos-20.09pre217446.ddf87fb1baf, home-manager"`
 - channels(felschr): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: keybase-gui
# a list of nixos modules affected by the problem
module:
```

**Comment by heronhaye** [comment]
Getting a fix out soonish.

**Comment by Currrupted** [comment]
For me this issue was resolved with the [latest nightly](https://github.com/keybase/client/issues/23018#issuecomment-601504560).

**Comment by FelschR** [comment]
I can confirm that [v5.3.1](https://github.com/keybase/client/releases/tag/v5.3.1) is working for me again.

**Comment by nh2** [comment]
> I can confirm that [v5.3.1](https://github.com/keybase/client/releases/tag/v5.3.1) is working for me again.

PR with that: #83011

**Comment by nh2** [comment]
Should be fixed by the just-merged #83011.

Please reopen if not.
